site/index.html

```diff
@@ -5,22 +5,27 @@
 ---
 <div class="header">
    <div class=" container ">
-      <div class=" row ">
-         <div class=" span12 col-sm-12 hidden-xs"><br></div>
+      <div class="row">
+         <div class=" span12 col-sm-12 hidden-xs"><br><br></div>
+      </div>
+      <div class="row" style="display: flex; align-items: center">
```

Comment:
Here I did 3 changes:

1. I added additional `<br>`. There was one before, now it is two. I added this `<br>` and a couple of others below in some places in order to put the "get started" button close to the center of the page, and push Videos section closer to the bottom of it. I did this guided by the typical advice for designing a landing page -> you want your CTA to be the most attractive thing to click that is visible [above the fold](https://www.optimizely.com/optimization-glossary/above-the-fold/). In our case that is "getting started" button, potentially also purple "try" section. Videos section is still partially visible above the fold, but is a bit more down and therefore looks more like smth additional to explore than one of the main things to look at.

2. I closed the div that is Bootstrap row here, instead of letting it also capture the stuff below. One Bootstrap row is always organized as 12 columns, but it was misused here, because somebody first added an element that spans all 12 columns, and then below one more element that spans 6, and then one more that spans 6!
The way Bootstrap handles such situation result in ok result, but it is semantically incorrect. So I closed this div here, so it contains only one element of 12 columns, and then opened a new one below (that is the last line). This element of 12 columns is anyway used to add vertical empty space in the whole width of the page, so it makes perfect sense that it is a standalone row.
This is merely a Bootstrap fix of the previous mistake somebody did.

3. In the new "row" I created, I used the opportunity to align Haskell logo with the code example to the right by their centers. Before they were aligned by their tops. That wasn't that much of a bit deal before when there was no "get started" button, but now with that button added, height of haskell logo vs code example + get-started button has bigger difference than before, which resulted in a bit weird look in my opinion.